### PR TITLE
Added image to home page & about page

### PR DESCRIPTION
**Pull request by wander-igdst**
## Description

Added image to home page & about page
## Related Issue
Add hero image to home page #26  

## How Has This Been Tested?
Tested and it is responsive

## Screenshots (if appropriate):
Screen Shot 2019-10-06 at 11 28 20 PM
Screen Shot 2019-10-06 at 11 28 06 PM

Screen Shot 2019-10-06 at 11 34 27 PM

## Checklist:

- [x] My code follows the code style of this project.
- [] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [] All new and existing tests passed.

### CODE REVIEW

**Comment by wander-igdst** [comment]
Screen Shot 2019-10-06 at 11 34 27 PM

File: next.config.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-      fs: 'empty'
-    }
-    return config
+      fs: "empty"
+    };
+    config.module.rules.push({
+      test: /\.svg$/,
+      use: ["@svgr/webpack"]
+    });
+    return config;
```
Comment (SchaeStewart) [inline next.config.js]: This isn't necessary. Next.js handles loading images.

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.8",
     "@fortawesome/free-solid-svg-icons": "^5.5.0",
     "@fortawesome/react-fontawesome": "^0.1.3",
+    "@svgr/webpack": "^4.3.3",
```
Comment (SchaeStewart) [inline package.json]: Also not necessary. Also, please refrain from adding packages without asking first. I prefer to keep the package list small.

**Review by SchaeStewart — CHANGES_REQUESTED** [review]
Thank you so much for contributing! I Made some comments in the review and I am asking you to make some changes before I approve. If you do not want to make changes or do not have time that's fine. I am happy to reassign the work.

Mainly I would prefer we use .png files instead of SVGs. I realize you may have not had access to the pngs and that is my fault. I'm attaching the png for the homepage here. I need some time to get the png for the about page.

Washington-DC-Capitol

thank you again for your contribution. If you have additional questions please ask here or on [slack](https://slack.codeforgreensboro.org)

File: next.config.js (hunk 1 above)
Comment (wander-igdst) [reply next.config.js]: That's right, sure!

File: package.json (hunk 2 above)
Comment (wander-igdst) [reply package.json]: Makes sense, removing the same

**Comment by wander-igdst** [comment]
Hello SchaeStewart! 

Thanks for taking time to review. Is there any specific reason to go with png? Because I could see noticeable difference in quality with either. Let me know how to proceed.

SVG:
Screen Shot 2019-10-07 at 12 10 04 PM

PNG:
Screen Shot 2019-10-07 at 12 09 55 PM

**Comment by SchaeStewart** [comment]
@wander-igdst The `.png` file size seems to be a little smaller and I would prefer to use the `.png` because it doesn't require adding an npm package.

**Comment by wander-igdst** [comment]
@SchaeStewart Updated the png format for the image on home page, thanks!

**Comment by SchaeStewart** [comment]
Thanks!

**Pull request merged**